src/ServicePulse.Host/vue/vitest.config.ts

```diff
@@ -1,6 +1,11 @@
 import { defineConfig } from "vitest/config";
 import viteConfig from "./vite.config.js";
 import path from "path";
+import { fileURLToPath } from "node:url";
+
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
These are not used anywhere in the file?

Reply:
**__filename** it is used in line 7: const __dirname = path.dirname(**__filename**);
__dirname is used later in line 18
 replacement: path.resolve(**__dirname**, './src'),

Reply:
Running `vue-tsc --build` results in error if this was not addressed

<img width="857" alt="image" src="https://github.com/Particular/ServicePulse/assets/545695/9693b613-c740-4d66-8e4a-b0ebea51d69e">

Reply:
Yeah, I noticed the same, it builds fine. It was annoying seeing it every time I used vue-tsc --build so that is why I pushed that change

Reply:
This is picked up from Node automatically, we don't need to define them